infoschema/cluster.go

```diff
@@ -58,6 +62,8 @@ var memTableToClusterTables = map[string]string{
 	TableTiDBTrx:                  ClusterTableTiDBTrx,
 	TableDeadlocks:                ClusterTableDeadlocks,
 	TableTrxSummary:               ClusterTableTrxSummary,
+	TableMemoryUsage:              ClusterTableMemoryUsage,
```

Comment:
Is there a test to cover the basic 
"select * from MEMORY_USAGE" or from TableMemoryUsageOpsHistory?

ClusterTableMemoryXXX might be harder, but I think we can cover the table with non-cluster-perfix